braindecode/datautil/transforms.py

```diff
@@ -0,0 +1,115 @@
+"""Transforms that work on Raw or Epochs objects.
+ToDo: should transformer also transform y (e.g. cutting continuous labelled 
+      data)?
+"""
+
+# Authors: Hubert Banville <hubert.jbanville@gmail.com>
+#          Lukas Gemein <l.gemein@gmail.com>
+#          Simon Brandt <simonbrandt@protonmail.com>
+#          David Sabbagh <dav.sabbagh@gmail.com>
+#
+# License: BSD (3-clause)
+
+import mne
+
+
+class FilterRaw(object):
+    """Apply mne filter on raw data
+
+    Parameters
+    ----------
+    mne_filter_kwargs : **kwargs
+        kwargs passed to mne.io.Raw.filter
+    """
+
+    def __init__(self, **mne_filter_kwargs):
+        self.mne_filter_kwargs = mne_filter_kwargs
+
+    def __call__(self, raw):
+        """Apply filter
+        
+        Parameters
+        ----------
+        raw : mne.io.Raw
+            raw data to filter
+        """
+        return raw.filter(**self.mne_filter_kwargs)
+
+
+class ZscoreRaw(object):
+    """Zscore raw data channel wise
+    """
+
+    def __call__(self, raw):
+        """Zscore Normalize raw data channel wise
+
+        Parameters
+        ----------
+        raw : mne.io.Raw
+            raw data to normalize
+
+        Returns
+        -------
+        raw : mne.io.Raw
+            normalized raw data
+
+        """
+        raw = raw.apply_function(lambda x: x - x.mean())
+        return raw.apply_function(lambda x: x / x.std())
+
+
+class FilterWindow(object):
```

Comment:
Note: @agramfort had suggested merging the two Filter objects and the two Zscore objects in #32

Reply:
I think we also said that they do not have to (should not) follow scikit-lean API as it is unnecessary. Also, we said they should be "smart" in a sense that they know whether they work on continuous, raw or epoched data.